src/main/java/org/checkerframework/specimin/PrunerVisitor.java

```diff
@@ -206,28 +208,17 @@ public Visitable visit(FieldDeclaration fieldDecl, Void p) {
   private Expression getBasicInitializer(Type fieldType) {
     if (fieldType.isPrimitiveType()) {
       PrimitiveType.Primitive primitiveType = ((PrimitiveType) fieldType).getType();
-      switch (primitiveType) {
-        case BOOLEAN:
-          return new BooleanLiteralExpr(false);
-        case INT:
-          return new IntegerLiteralExpr("0");
-        case LONG:
-          return new LongLiteralExpr("0L");
-        case FLOAT:
-          return new DoubleLiteralExpr("0.0f");
-        case DOUBLE:
-          return new DoubleLiteralExpr("0.0");
-        case BYTE:
-          return new IntegerLiteralExpr("0");
-        case SHORT:
-          return new IntegerLiteralExpr("0");
-        case CHAR:
-          return new CharLiteralExpr("'\u0000'");
-        default:
-          throw new RuntimeException("Unexpected primitive type: " + fieldType);
-      }
-    } else {
-      return new NullLiteralExpr();
+      return switch (primitiveType) {
+        case BOOLEAN -> new BooleanLiteralExpr(false);
```

Comment:
CI is failing because google-java-format is expecting Java 11 code. There are ways to work around it, but I'd prefer to be more portable and just stick to Java 11 features only: I'd like Specimin to run on as many JDKs as possible, since that makes life easier for all of us. (Switch expressions are a Java 14 feature, so that means we can't use them in Specimin if its language level is 11, which it is.)

Reply:
Ahhh, I see. Thanks

Reply:
reverting